src/ctap/storage.rs

```diff
@@ -415,17 +417,19 @@ impl PersistentStore {
             }
             Some((index, entry)) => {
                 debug_assert_eq!(entry.data.len(), 1);
-                let new_value = entry.data[0].saturating_sub(1);
-                self.store
-                    .replace(
-                        index,
-                        StoreEntry {
-                            tag: PIN_RETRIES,
-                            data: &[new_value],
-                            sensitive: false,
-                        },
-                    )
-                    .unwrap();
+                if entry.data[0] > 0 {
```

Comment:
To avoid extra-indentation, you can use:

```rust
if entry.data[0] == 0 {
    return;
}
```

Reply:
Done.